content/spin/spin-oci.md

```diff
@@ -16,9 +16,9 @@ url = "https://github.com/fermyon/developer/blob/main/content/spin/spin-oci.md"
 - [Conclusion](#conclusion)
 - [Next Steps](#next-steps)
 
-## Spin Open Container Initiative (OCI) Support
+## Spin Registry Support
 
-With Spin's Open Container Initiative (OCI) support, you can package and save your Spin application as an OCI artifact in a registry like [GitHub Container Registry (GHCR)](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry) or [DockerHub](https://hub.docker.com/) and then run your Spin app from these registries.
+With Spin's registry support, you can package and save your Spin application as an artifact in a registry like [GitHub Container Registry (GHCR)](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry) or [DockerHub](https://hub.docker.com/) and then run your Spin app from these registries.
```

Comment:
It would probably be good to mention what _kind_ of registries are supported, e.g. you can't store them in the Windows Registry _grin_

```suggestion
With Spin's registry support, you can package and save your Spin application as an artifact in an [OCI](https://opencontainers.org/about/overview/)-compliant registry like [GitHub Container Registry (GHCR)](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry) or [DockerHub](https://hub.docker.com/) and then run your Spin app from these registries.
```